MsWheaPkg/MsWheaReport/MsWheaEarlyStorageMgr.c

```diff
@@ -224,12 +224,20 @@ MsWheaESContentChangeChecksumHelper (
 {
   UINT16                        Sum16;
   MS_WHEA_EARLY_STORAGE_HEADER  Header;
+  EFI_STATUS                    Status;
 
   DEBUG ((DEBUG_INFO, "Calculate sum content helper...\n"));
 
   MsWheaESReadHeader (&Header);
   Header.ActiveRange += (UINT32)Length;
   MsWheaESCalculateChecksum16 (&Header, &Sum16);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_ERROR, "Checksum calculator failed - %r...", Status));

```

Comment:
I think this would be a little easier to understand in the context of a DEBUG message.


```suggestion
    DEBUG ((DEBUG_ERROR, "WHEA Early Storage Checksum calculation failed - %r...", Status));
```

Reply:
Updated.